src/index.html

```diff
@@ -1,13 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--1">
+      <div class="stars__star active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--2">
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--3">
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--4">
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star"></div>
+    </div>
+    <div class="stars stars--5">
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+      <div class="stars__star active"></div>
+    </div>
```

Comment:
```suggestion
    <div class="stars stars--5">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```
check comment below and remove all active classes.
they are redundant
for amount of active stars should be responsible stars--n modifier

Reply:
in js you will be this approach, no you will be use classNames and use class active, why not use this so

Reply:
because it is part of task description for learning modifiers and nth-child selector.
![Screenshot from 2023-04-02 18-01-13](https://user-images.githubusercontent.com/33029864/229361494-6146b325-a48e-4f07-96fd-8d306e6e6dc9.png)

also in js (react) in real world from server we can receive amount of active cards and can make dynamically modifier class which will be responsible for amount of active cards.
it is very common and more flexible approach to make parent responsible for child view.

Reply:
я убрал уже, что еще не так